web/src/js/ducks/settings.js

```diff
@@ -0,0 +1,92 @@
+import { StoreCmds } from '../actions'
+import { addLogEntry } from './eventLog'
+
+export const WS_MSG_TYPE = 'settings'
+export const WS_MSG_CMD_RESET = 'reset'
+export const WS_MSG_CMD_UPDATE = 'update'
```

Comment:
I agree on this. My point was to merge `WS_MSG_CMD_RESET` and `RESET` and `WS_MSG_CMD_UPDATE` and `UPDATE`.